src/js/requestHandlers.js

```diff
@@ -181,14 +192,14 @@ fluid.defaults("sjrk.storyTelling.server.deleteStoryHandler", {
     invokers: {
         handleRequest: {
             funcName: "sjrk.storyTelling.server.handleDeleteStory",
-            args: ["{arguments}.0", "{server}.deleteStoryDataSource", "{server}.storyDataSource"]
+            args: ["{arguments}.0", "{server}.deleteStoryDataSource", "{server}.storyDataSource", "{server}.options.globalConfig"]
```

Comment:
I believe I've achieved what you wanted. I'm still a little unsure about passing in the `request` object into `sjrk.storyTelling.server.handleDeleteStory` instead of passing the story ID and the success and error events separately, so I've left that part as it was. Is that more or less what you were thinking?

Reply:
Sorry, missed this question. Yes, handleDeleteStory() needs the pass in of `request` object for firing success and error events. Only passing the story id is not enough. Your change looks good.